runtime/compiler/env/J9ClassEnv.hpp

```diff
@@ -121,6 +121,22 @@ class OMR_EXTENSIBLE ClassEnv : public OMR::ClassEnvConnector
     */
    bool classHasIdentity(TR_OpaqueClassBlock *clazz);
 
+   /** \brief
+    *    Checks whether a class supports direct memory comparison if its fields meet
+    *    the criteria that NO field is of
+    *    - type double (D) or float (F)
+    *    - nullable-class/interface type (L)
+    *    - null-free class type (Q) that are not both flattened and recursively
+    *      compatible for the fast substitutability optimization
```

Comment:
Just one lingering reference to "the fast substitutability optimization" here - should say "direct memory comparison"

Reply:
Addressed in 68baa79